doc/nrf/known_issues.rst

```diff
@@ -978,9 +982,10 @@ KRKNWK-9214: Pigweed submodule might not be accessible from some regions
 HomeKit
 =======
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+.. rst-class:: v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
 
 KRKNWK-13010: Dropping from Thread to Bluetooth LE takes too long
+  Due to incorrect specification requirements, dropping from Thread to Bluetooth LE takes too long time and no Bluetooth connection can be established.
```

Comment:
```suggestion
  Dropping from Thread to Bluetooth LE takes much longer for FTD accessories than estimated in TCT030 test scenario.
  An accessory takes few minutes to detect a lack of network connection and then returns to Bluetooth LE within the required 65 seconds.
```

Reply:
Discussed with @mariuszpos . @mariusz-nordicsemi Could you please confirm if this is correct? Thank you!

Reply:
Adjusted as per suggestions from @mariusz-nordicsemi and @mariuszpos